packages/node/src/app/analytics-node.ts

```diff
@@ -91,7 +91,7 @@ export class Analytics
         path: settings.path,
         maxAttempts: settings.maxAttempts ?? 4,
         maxEventsInBatch: settings.maxEventsInBatch ?? 15,
-        maxWaitTimeInMs: settings.maxWaitTimeInMs ?? 1000,
+        flushInterval: settings.flushInterval ?? 1000,
```

Comment:
It doesn't look like the default value changed.

Reply:
We can talk offline! I was thinking about changing the default to something in the middle...